protocol/x/clob/keeper/process_operations_test.go

```diff
@@ -1912,7 +1912,28 @@ func setupNewMockEventManager(
 				),
 			).Once().Return()
 		}
+		if isClobMatchPerpetualDeleveraging(operation) {
+			mockIndexerEventManager.On("AddTxnEvent",
+				mock.Anything,
+				indexerevents.SubtypeDeleveraging,
+				indexerevents.DeleveragingEventVersion,
+				mock.Anything,
```

Comment:
nit: Is it possible to derive the deleveraging event here from the operation?

Reply:
Bankruptcy price isn't exposed by the API. I could add it to the test params, but that would require some more changes. Plus, the exact deleveraging event is being tested in multiple other e2e tests. So, I think it's reasonable to omit here.

Reply:
Sounds good, can we add a comment to that effect?

Reply:
done